software-engineering-policies/TechnicalDebt/TechnicalDebtPolicy.md

```diff
@@ -1,48 +1,168 @@
+
+Ideally even if technical debt is going to be resolved immediatly raising it is good for understanding its impact on UKHO work and identifying that technical debt was discovered.
+
+When raising Technical Debt, please add the following information to the Technical Debt PBI:
+
+### Title
+
+```Technical Debt - {Concise outline of Technical Debt item}```
+
+### Description
+
+The description of the Technical Debt item must include the following as a minimum:
+
+* The repository the Technical Debt lives within (a link to the repo would be good but the name would suffice)
+* Description of the issue in as much detail as you can provide, including class/file names if this can be included
+* A proposed solution or ideas for a fix **
+* Cost/Benefit and risk of the Technical Debt item, simply as a "why should this be fixed?/How long would it take?"
+
+### Tagging Strategy
+
+* The Technical Debt PBI **MUST** be tagged with the `````Technical Debt````` tag and should have any further appropriate tags, such as the following:
+  * Product Name (could be multiple, see the product list in AppRegister; http://appregister/)
+  * Tech Debt Type:
+    * Security
+    * Knowledge
+    * Automation
+    * Testing
+    * Architecture
+    * Code
+    * Infrastructure
+  * Tech Debt Impact:
+    * Availability
+    * Portability
+    * Compatibility
+    * Performance
+    * Maintainability
+    * Extensibility  
+  * Out-Of-Support**
+  * Investigate To Remove** - This tag has been used to mark a Technical Debt item that may already have been completed or is going to be irrelevant due to other work.
+  * UKHO TD priority tag:
+    * TD1 - High business/technical value, high risk debt that needs to be paid off ASAP (a Security tag should automatically be considered for a TD1 prioritisation).
+    * TD2 - High business/technical value (cost reduction, blocker removing, maintainability improvement), lower risk, a change that should be worked on when time/opportunity permits and is not something that can be accepted or supported long term.
+    * TD3 - Tech debt that has been accepted as a risk but through paying off would add value through improving usability, maintainability, reliability or performance.
+    * TD4 - Accepted risk from the business, safe to leave until service reaches end of life. Worth tracking in case developers are working in the area and can complete as quick wins.
+
+** - Denotes where the tag is appropriate
+
+### Tech Debt Metric Strategy
+
+Teams **MUST** adopt the use of the TD metrics section in their PBI template. This enables the business to gather much more useful and accurate information about technical debt and empowers the automations available for tracking and monitoring. For support in adding this section to your PBI template, either request DDC to implement the change to your templates or discuss with your delivery manager.
```

Comment:
Although this sentence says teams **must** use TD metrics, the screenshot later on marks it as optional. This is confusing.

Reply:
Would love an alternative if you have one, it is a case of MUST use it when recording TD but only useful when recording technical debt. So happy to have recommendations

Reply:
Could have a separate type of work item, a custom one?

Reply:
You could also have a blank template with what you need to fill in. People don't read policy so imbedding it in the process will make it more likely to happen.

Reply:
How about naming the section in the PBI template
`Tech Debt Metrics (if applicable)`
or
`Tech Debt Metrics (Tech Debt only)`

Reply:
> Could have a separate type of work item, a custom one?

In regards to creating a "TD" work item, it is possible but may require the complete rewrite of portions of the automations in place now. for minimal work and maximum benefit I think Sara's point of a better name for the PBI section would be better.

Reply:
Updated the Image with Sara's recommendation